docs/UserGuide.md

```diff
@@ -3,271 +3,443 @@ layout: page
+**Purpose**: Deletes the specified client from ClientBook after listing or finding contacts.
 
-* Selects the client at the specified `INDEX`.
+**Format**: `delete INDEX`
+
+* Deletes the client at the specified `INDEX`.
 * The index refers to the index number shown in the displayed client list.
 * The index **must be a positive integer** 1, 2, 3, …​
-* Policy ID cannot contain '>' character!
-* Note that your URLs with should not contain '>' characters either! They are not valid URLs by the Internet's definition.
 
-Examples:
-* `list` followed by `policy 2` displays the policies associated with the 2nd person in the address book.
-* `find Betsy` followed by `policy 1` displays the policies associated with the 1st person in the results of the `find` command.
+**Examples**:
+* `list` followed by `delete 2` deletes the 2nd person in the address book.
 
-### Sorting the list of clients : `sort`
+  ![delete 2](images/delete-2.png)
 
-Sorts the list of clients in ClientBook.
+* `find n/Charlotte` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-Format: `sort ATTRIBUTE DIRECTION`
+  ![delete charlotte](images/delete-charlotte.png)
 
-* Sorts the list of clients according to the specified `ATTRIBUTE` and `DIRECTION`.
-* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy.
-* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order.
 
-Examples:
-* `sort -n -asc` sorts the list of clients by name in ascending order.
+### Sort list of clients: `sort`
 
-### Exiting the program : `exit`
+**Purpose**: Sorts the current list of clients in ClientBook.
+
+**Format**: `sort -ATTRIBUTE -DIRECTION`
 
-Exits the program.
+* Sorts the list of clients according to the specified `ATTRIBUTE` and `DIRECTION`.
+* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy, but not both.
+* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order, but not both.
+
+**Examples**:
+* Sort the current list of clients by name in ascending order.
+  * `sort -n -asc`
+
+    ![sort](images/sort.png)
 
-Format: `exit`
 
-### Locking ClientBook : `lock`
+### Lock ClientBook with a user-selected password: `lock`
 
-Locks ClientBook.
+**Purpose**: Locks ClientBook with a user-selected password.
 
-Format: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
+**Format**: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
 
 * Verifies the current password before locking ClientBook with the new password.
 * The `CURRENT_PASSWORD` field can be omitted if ClientBook is not yet locked.
 * When `CURRENT_PASSWORD` and `NEW_PASSWORD` fields are both omitted, ClientBook will attempt to lock itself using the last used password that is safely stored on your device.
 
-### Unlocking ClientBook : `unlock`
+:exclaimation: **Note**: After setting a password, application can only be launched through Command Prompt or Terminal.
```

Comment:
I think there is a typo with "exclamation".

Reply:
Thanks for spotting this!